client.go

```diff
@@ -491,6 +491,20 @@ func (client *gocloak) UpdateUser(token string, realm string, user User) error {
 	return nil
 }
 
+// UserGroupJoin joins User into a Group
+func (client *gocloak) UserGroupJoin(token string, realm string, userGroupLink UserGroupLink) error {
+	resp, err := getRequestWithBearerAuth(token).
+		SetBody(userGroupLink).
+		Put(client.getAdminRealmURL(realm, "users", userGroupLink.UserID, "groups", userGroupLink.GroupID))
+
+	err = checkForError(resp, err)
+	if err != nil {
```

Comment:
Proponuję do jednej linijki. :)

Reply:
@kazmier nu nu, Panowie (autorzy) tak piszą więc nie chcę zmieniać stylu

Reply:
@plsip Panowie autorzy czego? To jakiś plugin jest?

Reply:
A, ok, gocloak. Daję _approve_, then.